streams-designer/README.md

```diff
@@ -0,0 +1,155 @@
+1. Drag and drop the `Code` operator under **Sources** on to the canvas.
+1. Select the `Code` operator and a right pane will open for editing the code.
+![Code operator in Streams Designer](code-operator-streams-designer.png)
+1. Select `Python 3.5` as the coding language and replace what's existing with the code below.<br/>
+   **Note:** Add your Cloudant credentials as parameters in the Cloudant initializer `client = Cloudant(...)`.
+    ```python
+    import sys
+    from cloudant import Cloudant
+    from cloudant.feed import Feed
+    
+    # init() function will be called once on pipeline initialization
+    # @state a Python dictionary object for keeping state. The state object is passed to the produce function
+    
+    
+    def init(state):
+        # do something once on pipeline initialization and save in the state object
+        pass
+    
+    
+    # produce() function will be called when the job starts to run.
+    # It is called on a background thread, and it will typically invoke the 'submit()' callback
+    # whenever a tuple of data is ready to be emitted from this operator.
+    # This allows for using asynchronous data services as well as synchronous data generation or retrieval.
+    # @submit a Python callback function that takes one argument: a dictionary representing a single tuple.
+    # @state a Python dictionary object for keeping state
+    # You must declare all output attributes in the Edit Schema window.
+    def produce(submit, state):
+        # Replace with your Cloudant credentials
+        client = Cloudant(
+            'username',
+            'password',
+            account='account',
+            connect=True
+        )
+        db = client['animaldb']
+        feed = Feed(db, feed='continuous', include_docs=True)
+        for change in feed:
+            doc = change['doc']
+            if 'payload' in doc:
+                output_doc = {}
+                output_doc['_id'] = doc['_id']
+                output_doc['_rev'] = doc['_rev']
+                if 'wiki_page' in doc['payload']:
+                    output_doc['wiki_page'] = doc['payload']['wiki_page']
+                if 'min_weight' in doc['payload']:
+                    output_doc['min_weight'] = doc['payload']['min_weight']
+                if 'max_weight' in doc['payload']:
+                    output_doc['max_weight'] = doc['payload']['max_weight']
+                if 'class' in doc['payload']:
+                    output_doc['class'] = doc['payload']['class']
+                if 'diet' in doc['payload']:
+                    output_doc['diet'] = doc['payload']['diet']
+                # Submit a tuple in each iteration:
+                submit(output_doc)
+    ```
+1. Select the `Edit Output Schema` below the code editor.
+1. Add the following attribute names and their types using the **Add Attribute** button.
+   Press **Save** then **Close** when finished.
+
+    Name | Type
+    --- | --- 
+    _id | Text
```

Comment:
`\_` to escape the `_` for this and rev
Whilst it is rendering correctly when viewed on GH in the editor all the following content appears italicized.